webapp/channels/src/components/admin_console/brand_image_setting/brand_image_setting.test.tsx

```diff
@@ -26,21 +19,68 @@ describe('components/admin_console/brand_image_setting', () => {
         unRegisterSaveAction: jest.fn(),
     };
 
-    test('should have called deleteBrandImage or uploadBrandImage on save depending on component state', () => {
-        const wrapper = shallow<BrandImageSetting>(
-            <BrandImageSetting {...baseProps}/>,
-        );
+    const deleteButtonTestId = 'remove-image__btn';
+
+    /**
+     * The previous test directly called 'handleSave' to test the 'deleteBrandImage' and 'uploadBrandImage' functions.
+     *
+     * This is only possible with a class component and Enzyme, but RTL promotes testing components the way
+     * a user would interact with them, hence the updated tests.
+     *
+     * 'deleteBrandImage' and 'uploadBrandImage' are only accessible in the 'handleSave' method which can't be
+     * invoked via any user interactions.
```

Comment:
I'm not sure about this... is there a way to test handleSave? Doing so would maybe catch the bug that I suspect is here.